test/regress/tests/console/expected/add.out

```diff
@@ -5,19 +5,23 @@
 	You can find documentation here 
 https://github.com/pg-sharding/spqr/tree/master/docs
 
+CREATE KEY RANGE krid1 FROM 1 ROUTE TO sh1;
+ERROR:  you didn't have any distributions.
 CREATE DISTRIBUTION ds1 COLUMN TYPES integer;
     add distribution    
 ------------------------
  distribution id -> ds1
 (1 row)
 
-CREATE KEY RANGE krid1 FROM 1 ROUTE TO sh1 FOR DISTRIBUTION ds1;
```

Comment:
Consider adding new tests rather than rewriting existing

Reply:
💯

Reply:
I changed the logic of creating the key range. We have a test testing the logic of creating key range and distribution. I'm changing it without doing a duplicate test. If you think you need to do a separate test, then I will do a separate key_range test.